clang/lib/DPCT/APINamesTemplateType.inc

```diff
@@ -58,94 +58,169 @@ TYPE_REWRITE_ENTRY("cub::TransformInputIterator",
                        HeaderType::HT_DPL_Iterator,
                        TYPE_FACTORY(STR("oneapi::dpl::transform_iterator"),
                                     TEMPLATE_ARG(2), TEMPLATE_ARG(1))))
-
+// cub::ConstantInputIterator
 TYPE_REWRITE_ENTRY("cub::ConstantInputIterator",
-                   HEADER_INSERTION_FACTORY(HeaderType::HT_DPCT_DPL_Utils,
-                   TYPE_FACTORY(STR(MapNames::getLibraryHelperNamespace() + "constant_iterator"), TEMPLATE_ARG(0))))
+                   HEADER_INSERTION_FACTORY(
```

Comment:
You don't need to emit a warning when using dpl helper functions.

Reply:
Done